### ISSUE TITLE
Notes / History handling should be less dangerous

### ISSUE THREAD

**Issue by ralfbiedert** [hedgedoc/hedgedoc#1823]
### Which part of the project should be enhanced?

The "History" page, esp. the "Clear history" button.

### Is your enhancement request related to a problem? Please describe.

Like the commenter in #1199 I set up my own instance and started using it for a bit. I, too, pressed "Clear history" to discover that all my notes have been deleted. 

I think the History panel, as well as the main "dashboard" (i.e., the root site when accessing my instance) would benefit from a clearer persistence model, better button labeling and better warnings.

### Describe the solution you'd like

The absolute MVP should be renaming "Clear history" to "Delete all my documents".

Even better, since I create 100's documents a year which I was hoping to eventually do in HedgeDoc, I'd like to see the "History" panel being replaced with a tidier "Documents" panel (I want to primarily manage documents, not history), which could have a "Recent" section.  In that panel can still be a "Clear" button, but ultimately an operation as dangerous as "Delete All" should be hidden somewhere in the options menu, similar to how Github does it:

![image](https://user-images.githubusercontent.com/301288/141862183-1bb60666-3724-411a-b3fd-a53209512029.png)

### Describe alternatives you've considered

As mentioned above, MVP is rename button and warning to "Delete all my documents".

**Comment by SISheogorath** [comment]
Please be aware that the Clear history button **does not delete your notes**. Your notes are still there, just the reference to them is gone. In order to delete your notes you have to either delete your account or delete each note individually. Therefore "clear history" does exactly what it says it does. It clears the history view.

**Comment by ErikMichelson** [comment]
This is the way how it's currently implemented for version 2.

![Screenshot_20211116_082445](https://user-images.githubusercontent.com/52606896/141939719-c027b650-2155-45ba-b9b7-f94ba36db708.png)

**Comment by ErikMichelson** [comment]
Regarding your idea for chaging from the history page to a "documents" page, I recommend this issue: #994

**Comment by ralfbiedert** [comment]
Ah, yes, I can confirm my notes were still there, found them via "Export user data". 

I came to realize I fundamentally misunderstood the mental model the app is built on.  In essence, I perceived it as a "Markdown document store that happens to allow for collaboration", while apparently it seems to be more of a "Multi-user sketchpad that happens to remember your notes". 

Having a feature to list all notes as discussed in #994 would probably have avoided my confusion. Also, I really like the screenshot you posted, that extra sentence is much clearer than the current one.

I'll close this issue since it seems everything is / will be resolved. 

Thanks for all the clarification!
